projects/storefrontapp-e2e-cypress/cypress/sample-data/checkout-flow.ts

```diff
@@ -95,3 +95,27 @@ export const cartWithCheapProduct: SampleCartProduct = {
   total: '$8.20',
   totalAndShipping: '$20.19',
 };
+
+export const carts: SampleCartProduct[] = [
+  {
+    estimatedShipping: '11.99',
```

Comment:
To make the test more robust, we try to avoid using hardcoded values. If you could instead, after adding the first item to cart, grab all the info you need from the response (product name, product Id or price), and assert against those instead.